### PR TITLE
Admin chores

### PR DESCRIPTION
**Pull request by AStoerkel1**
this PR completes [11964](https://app.shortcut.com/clarkcan/story/11964/admin-dashboard-chores)

### CODE REVIEW

**Review by pzalep1 — APPROVED** [review]

File: src/app/admin/components/filter-search/filter-search.component.ts [hunk 1]
```diff
@@ -248,19 +274,18 @@ export class FilterSearchComponent implements OnInit {
     }
   }
 
-  toggleTopicFilter(filter: any) {
+  toggleTopicFilter(filter: {name?: string, _id: string}) {
```
Comment (JosueMolinaMorales) [inline src/app/admin/components/filter-search/filter-search.component.ts:277]:
```suggestion
  toggleTopicFilter(filter: { name?: string, _id: string }) {
```

**Review by JosueMolinaMorales — COMMENTED** [review]
Looks good. Just a small thing

File: src/app/admin/components/learning-object-list-item/learning-object-list-item.component.html [hunk 2]
```diff
@@ -35,20 +35,22 @@
...
       <li (click)="toggleRelevancyDate(true)"><i class="fal fa-history"></i>Mark Next Check Date</li>
       <li (click)="toggleAddEvaluatorModal(true)"><i class="fal fa-user-plus"></i>Assign Evaluators</li>
       <li *ngIf="hasParents === false" (click)="toggleChangeAuthorModal(true)"><i class="far fa-user"></i>Change Author</li>
       <li *ngIf="learningObject.status !== 'released' && (auth.user.accessGroups.includes('admin') || auth.user.accessGroups.includes('editor'))" (click)="toggleChangeCollectionModal(true)"><i class="fal fa-exchange"></i>Change Collection</li>
       <li *ngIf="learningObject.status === 'released' && (auth.user.accessGroups.includes('admin') || auth.user.accessGroups.includes('editor'))" (click)="toggleUnreleaseConfirm(true)"><i class="far fa-eye-slash"></i>Unrelease</li>
-      <li class="new-feature_wrapper" [routerLink]="['/onion/relevancy-builder', learningObject.id]">
-        <p class="list-item_title"><i class="far fa-puzzle-piece"></i>Map & Tag</p>
-        <p class="new-item_element"><i class="fas fa-asterisk"></i>NEW</p>
-      </li>
+      <a [routerLink]="['/onion/relevancy-builder', learningObject.id]" target="_blank">
```
Comment (mfranz1) [inline src/app/admin/components/learning-object-list-item/learning-object-list-item.component.html:48]: What is this component you're linking to?

File: src/app/admin/components/filter-search/filter-search.component.ts [hunk 3]
```diff
@@ -72,13 +74,37 @@ export class FilterSearchComponent implements OnInit {
...
+        for(const status of qParams.status) {
+          this.toggleStatusFilter(status);
+        }
+      } else {
+        this.toggleStatusFilter(qParams.status);
+      }
+    }
+    if(qParams.collection){
+      this.toggleCollectionFilter(qParams.collection);
+    }
```
Comment (mfranz1) [inline src/app/admin/components/filter-search/filter-search.component.ts:100]: Could you add a few more comments just describing the if statements above ^^

**Review by pmanal1 — APPROVED** [review]

File: src/app/admin/components/filter-search/filter-search.component.ts (hunk 3 above)
Comment (AStoerkel1) [reply src/app/admin/components/filter-search/filter-search.component.ts:108]: gotcha

**Review by EVogel1999 — APPROVED** [review]

**Pull request merged**